docs/data/charts/label/label.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Charts - Label
+productId: x-charts
+components: BarChart, ScatterChart, LineChart, PieChart
+---
+
+# Charts - Label
+
+<p class="description">Label refers to the text reference of a series or data. It is used in many components like the "legend", "tooltip",  and inside an "arc" of a pie chart.</p>
+
+## Basic display
+
+In the most basic example, you can pass in a `string` as a series' label, and it will be rendered like that by all the different components that use it in order to differentiate each series.
```

Comment:
Not sure we need to introduce what a label is. We can assume it's common knowledge.


```suggestion
To set series' label, you can pass in a `string` as a series' property `label`.
The provided label will be visible at different locations such as the legend, or the tooltip.

:::info
The Pie chart has some specificity described in its [own section](#pie).
:::
```